test/repository/ci-workflows.test.ts

```
@@ -103,6 +103,15 @@ describe('CI workflows', () => {
     expect(workflow).toContain('subjectPattern: ^.{1,72}$');
   });
 
+  it('greets first-time contributors from the trusted base workflow', async () => {
+    const workflow = await readWorkflow('greeting-guideline-pr.yml');
+
+    expect(workflow).toMatch(/^  pull_request_target:$/m);
```

ESLint

[error] Spaces are hard to count. Use {2}.

(no-regex-spaces)